Update dependencies-and-umbrella-projects.markdown (#1677)
elixir-lang/elixir-lang.github.com: getting-started/mix-otp/dependencies-and-umbrella-projects.markdown

lockfile: "../../mix.lock",
```

> Those options mean all dependencies will be checked out to `kv_umbrella/deps`, and they will share the same build, config and lock files. We haven't talked about configuration yet, but from here we can get the idea that all configuration and dependencies are shared across all projects in an umbrella, and it is not per application.

Reviewer: ```suggestion
Those options mean all dependencies will be checked out to `kv_umbrella/deps`, and they will share the same build, config, and lock files. We haven't talked about configuration yet, but keep in mind all configuration and dependencies are shared across all projects in an umbrella, and it is not per application.
```
